minivm.S

```diff
@@ -311,7 +311,8 @@ MINIVM_handle_tlbmissrw:
 
 	r30.h = #HI(PAGE_TABLE_VA)	// form l2 vaddr
 
-	tlbw(r29:28, r3)
```

Comment:
Interesting, what r3 was representing before here? It doesn't seem to be set anywhere in this function, and I don't think it would be an missrw argument, right?